lldb/source/Plugins/SymbolFile/DWARF/DWARFBaseDIE.h

```diff
@@ -95,6 +100,8 @@ class DWARFBaseDIE {
 
   const char *GetName() const;
 
+  const char *getShortName() const { return GetName(); }
```

Comment:
Nit: How do people feel about grouping all the new APIs that are required for LLVM compatbility together and commenting on adding a comment stating why we have them?

Reply:
(I was considering the same thing, but I don't particularly care either way)